gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -40,17 +40,16 @@
 _TOTAL_BASE_PAIRS_SIG_DIGITS = 4
 _PARTITION_SIZE_SIG_DIGITS = 1
 
+START_POSITION_COLUMN = 'start_position'
```

Comment:
s/START_POSITION_COLUMN/START_POSITION_COLUMN_NAME

Reply:
I realized we have `ColumnKeyConstants` so there is no need for this const value.